types/api.go

```diff
@@ -54,8 +54,18 @@ func (b *BaseAPI) SetPosition(pos Position) {
 	b.Position = pos
 }
 
+// Stop the api
 func (b *BaseAPI) Stop() {
 	log.Printf("Stopping api %s (%s)\n", b.Name, b.UUID)
 	b.StopChan <- true
 	log.Printf("Done stopping!\n")
 }
+
+// Send to websocket
+func (b *BaseAPI) Send(data interface{}, err error) {
```

Comment:
Regarding above comment, I would keep this the same and remove the SendErrorOrSuccess thing.

Reply:
I like the abstraction though. Makes sense for the API to have its own send method. That way we could change how an API sends data, and it wouldn't affect the rest of the code (hopefully) inside of the API package.